onnxruntime/test/platform/ios/ios_package_test/ios_package_testTests/ios_package_test_c_api.m

```diff
@@ -1,97 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-//
-//  ios_package_test_c_api.m
-//  ios_package_testTests
-//
-//  This file hosts the tests of ORT C API, for tests of ORT C++ API, please see ios_package_test_cpp_api.mm
-//
```

Comment:
Why do we need to delete this? if we can call the C API if it was compiled with the C++ compiler aren't these tests still valid?

Reply:
The C API is called by C++ API, which is tested in the C++ test already, do we want to maintain test for both C API and C++ API?

Reply:
OK. As it's doing the same test it's fine to drop.